### DISCUSSION TITLE
Vulnerability of dependency " github.com/blevesearch/bleve"

### DISCUSSION THREAD

**Discussion by Silence-worker-02** [git-bug/git-bug#1078] [Feature & Ideas]
Hello, we are a team researching the dependency management mechanism of Golang. During our analysis, we came across your project and noticed that it contains a vulnerability (CVE-2022-31022). In your project, the github.com/blevesearch/bleve package is being used at version github.com/blevesearch/bleve v1.0.14, but the patched version is v2.3.3. To fix the vulnerability, we recommend modifying the go.mod file to update the version to v2.3.3 or higher. Thank you for your attention to this matter.

**Comment by JoacoG** [comment]
Hello, this issue could be addressed? I don't see the change in the go.mod file

**Comment by sudoforge** [comment]
We do not use the `bleve/http` package, even indirectly, which is where this vulnerability exists. That said, it wouldn't hurt to upgrade the dependency.
